x/emissions/keeper/queryserver/query_server_stake.go

```diff
@@ -114,3 +115,80 @@ func (qs queryServer) GetTopicStake(ctx context.Context, req *types.QueryTopicSt
 
 	return &types.QueryTopicStakeResponse{Amount: stake}, nil
 }
+
+func (qs queryServer) GetStakeRemovalsForBlock(
+	ctx context.Context,
+	req *types.QueryStakeRemovalsForBlockRequest,
+) (*types.QueryStakeRemovalsForBlockResponse, error) {
+	removals, err := qs.k.GetStakeRemovalsForBlock(ctx, req.BlockHeight)
+	if err != nil {
+		return nil, err
+	}
+	maxLimit := types.DefaultParams().MaxPageLimit
+	moduleParams, err := qs.k.GetParams(ctx)
+	if err == nil {
+		maxLimit = moduleParams.MaxPageLimit
+	}
+	if uint64(len(removals)) > maxLimit {
+		return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("cannot query more than %d removals at once", maxLimit))
+	}
+	removalPointers := make([]*types.StakeRemovalInfo, len(removals))
+	for i, removal := range removals {
+		removalPointers[i] = &removal
+	}
+	return &types.QueryStakeRemovalsForBlockResponse{Removals: removalPointers}, nil
```

Comment:
so I'm with you that it's weird but this is how the autogenerated types come out of protobuf, since it's a complex type (struct StakeRemovalInfo) protobuf returns that into a pointer. When you actually query via the CLI it returns the data, though:

```yaml
$ allorad --home=test/devnet/validator0 query emissions stake-removals-for-block 362936

removals:
- amount: "7048718058408485432"
  block_removal_completed: "362936"
  block_removal_started: "56"
  reputer: allo1vja3ec4a707znzlzhs4pyqw8m5y98euwka5uf5
  topic_id: "1"
```

so I think not an issue